server/memphis_handlers_factories.go

```diff
@@ -164,6 +163,46 @@ func (fh FactoriesHandler) CreateFactory(c *gin.Context) {
 	})
 }
 
+var ErrFactoryAlreadyExists = errors.New("memphis: factory already exists")
```

Comment:
what soed it means direct?

Reply:
unlike HTTP requests, that go to the HTTP handler function and have the middleware's context (getUserDetailsFromMiddleware), SDK requests are processed directly. we had to differentiate between them somehow and this naming was chosen... we're open for other suggestions :)